coralproject/talk
# [CORL-478] Porting over Danish translations to v5

## What does this PR do?
- Adds Danish for v5

## How do I test this PR?

- Add LOCALE env variable "da"
- See copy reflected in Danish
- Corrections to typos and grammar welcome!
